packages/native/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "@emotion/native",
+	"version": "1.0.0",
+	"description": "Style and render React Native components using emotion",
+	"main": "dist/index.cjs.js",
+	"module": "dist/index.esm.js",
+	"files": ["src", "dist"],
+	"devDependencies": {
+		"emotion-theming": "^9.2.5",
+		"react": "^16.4.1",
+		"react-native": "0.46.0"
+	},
+	"dependencies": {
+		"@emotion/primitives-core": "1.0.0"
+	},
+	"peerDependencies": {
+		"prop-types": "15.x",
+		"react-native": "0.46.0"
```

Comment:
Not sure about this version, maybe it should be looser since iirc the latest version of react-native is 0.56

Reply:
For testing purposes, I fallback to 0.46.0. I'll update it!